pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -594,7 +597,13 @@ func (c *Controller) SetDenyConnStore(denyConnStore *connections.DenyConnectionS
 // Run will not return until stopCh is closed.
 func (c *Controller) Run(stopCh <-chan struct{}) {
 	attempts := 0
-	if err := wait.PollUntilContextCancel(wait.ContextForChannel(stopCh), 200*time.Millisecond, true, func(ctx context.Context) (bool, error) {
+	// If Antrea client is not ready within 5s, we assume that the Antrea Controller is not
+	// available. We proceed with our watches, which are likely to fail. In turn, this will
+	// trigger the fallback mechanism.
+	// 5s should be more than enough if the Antrea Controller is running correctly.
+	ctx, cancel := context.WithTimeout(wait.ContextForChannel(stopCh), 5*time.Second)
```

Comment:
@tnqn I tried a few things, but this was the simplest and most "correct" IMO.
Given that `ConfigMapCAController` is third-party code, it's hard to come up with a better solution (my preferred solution would have been to wait until both controllers have "synced" and processed initial items).
This works well in practice: if the antrea-controller is already running, the client should be ready with 1 or 2 seconds so we don't hit the timeout; otherwise, the timeout is short enough that we don't wait too long before falling back to local policy files.